atd-vzd/triggers/is_crash_in_austin_full_purpose.sql

```diff
@@ -0,0 +1,13 @@
+-- This function determines whether crash coordinates fall within the Austin Full Purpose jurisdiction polygon.
+-- If the crash has no coordinates, it returns whether the crash is in Austin (city_id = 5)
```

Comment:
it might be good to add a note here that this is invoked by a generated column function, in case that migration kind of gets buried over time.